### PR TITLE
Update blogs.json

### PR DESCRIPTION
**Pull request by priyankarpal**
Added new websites

image

### CODE REVIEW

File: database/resources/blogs.json [hunk 1]
```diff
@@ -60,5 +60,19 @@
         "url": "https://hashnode.com",
         "category": "resources",
         "subcategory": "blogs"
+    },
+    {
+        "name": "Omglinux",
+        "description": "A website where we can find LInux related blogs",
```
Comment (rupali-codes) [inline database/resources/blogs.json:66]: thanks Priyankar. could you please add a little more description for it?

Comment (CBID2) [inline database/resources/blogs.json:66]: You spelled Linux as "LInux". I think that should be fixed 😄

File: database/resources/blogs.json [hunk 2]
```diff
@@ -60,5 +60,19 @@
...
+        "url": "https://www.omglinux.com/",
+        "category": "resources",
+        "subcategory": "blogs"
+    },
+     {
+        "name": "Showwcase",
+        "description": "Showwcase is a professional network built for developers to connect, build community, and find new opportunities",
+        "url": "https://www.omglinux.com/",
```
Comment (CBID2) [inline database/resources/blogs.json:74]: I found the link for Showwcase. Here it is: https://www.showwcase.com/

**Comment by priyankarpal** [comment]
Resolved these errors.

**Comment by CBID2** [comment]
> Resolved these errors.

Great job @priyankarpal! 😁

**Review by CBID2 — APPROVED** [review]
LGTM! 😊

**Review by CBID2 — COMMENTED** [review]
👍🏽

**Review by rupali-codes — APPROVED** [review]
thanks priyankar

**Pull request merged**